pkg/config/config_windows.go

```diff
@@ -28,7 +28,11 @@ var (
 const ServiceName = "DatadogAgent"
 
 func osinit() {
-	pd, err := winutil.GetProgramDataDir()
```

Comment:
Is this correct? It seems like you would want this line.

Reply:
That was a test that I committed by mistake. Thanks for the catch !